Fix gulpfile instructions (#38280)
mdn/content: files/en-us/learn_web_development/extensions/testing/automated_testing/index.md

   Each gulp task is exported in the same basic format — `exports function taskName(cb) {...}`. Each function takes one parameter — a callback to run when the task is completed.

6. You can run your gulp's default task with the following command — try this now:

   ```bash
   gulp
   ```

Now we are ready to add more tasks to our Gulp file. Each addition may require you to modify the `gulpfile.mjs` file in the following way:

- If we ask you to add some `import` statements, add them below the existing `import` statement.
- If we ask you to add a new `export function ...` statement, add it to the end of the file.
- If we ask you to change the default export, change the `export default` statement in the way we specify.

So your `gulpfile.mjs` file will grow like this:

```js
import gulp from "gulp";
// Add any new imports here

// Our latest default export
// export default ...

// Add any new task exports here
// export function ...
// export function ...

> ```

Reviewer: Minimally though, I'd move this whole section under "Adding some real tasks to Gulp" and change "If we ask you to add " to "When we ask"

What I'd like you to do is put the instructions inline with where they are used instead. So
> Add the following dependency to gulpfile.mjs

Becomes

> Add the following dependency to `gulpfile.mjs` below the existing imports.

It's a bit more work for you, but it is a lot easier for readers to follow. That bit's up to you.

Author: The OP was confused not because of the import but because of how to replace the default export. While I'm not sure if the new syntax would be as confusing as the old one, I think this way is the most explicit.

Reviewer: I'm OK with it - certainly an improvement. But since are are definitely going to ask for change, perhaps when, not if:

```suggestion
Now we are ready to add more tasks to our Gulp file. Each addition may require you to modify the `gulpfile.mjs` file in the following way:

- When we ask you to add some `import` statements, add them below the existing `import` statement.
- When we ask you to add a new `export function ...` statement, add it to the end of the file.
- When we ask you to change the default export, change the `export default` statement in the way we specify.

So your `gulpfile.mjs` file will grow like this:

```js
import gulp from "gulp";
// Add any new imports here

// Our latest default export
// export default ...

// Add any new task exports here
// export function ...
// export function ...
```
```
